src/kayenta/report/list/table.tsx

```diff
@@ -22,43 +88,46 @@ const columns: ITableColumn<ICanaryExecutionStatusResult>[] = [
         application={execution.application}
       />
     ),
-    width: 1,
   },
   {
-    label: 'Score',
-    getContent: execution => (
-      <Score
-        score={execution.result.judgeResult.score}
-        inverse={true}
-        showClassification={false}
-      />
-    ),
-    width: 1,
-  },
-  {
-    label: 'Started',
-    getContent: execution => <FormattedDate dateIso={execution.startTimeIso}/>,
-    width: 1,
+    label: 'Scopes',
+    getContent: ({ canaryExecutionRequest: { scopes } }) => {
+      const baselineScopeNames = Object.keys(scopes).reduce<Set<string>>((acc, scope) =>
```

Comment:
I am shocked that you have to pass `Set<string>` as a type parameter to `reduce`. Is your preference to be explicit, or is the TS compiler dumb here? Or do you need to pass `new Set<string>()` as the second parameter to reduce?

Reply:
hah. I think I just automatically reach for typing `reduce` as a reflex from trying to type more convoluted uses of reduce (i.e. the accumulator goes through some type transitions prior to returning more explicitly typed value). I'll switch these over to the `Set` itself :D